wikipedia-sentences/pods/doc.yml

```diff
@@ -0,0 +1,17 @@
+!CompoundIndexer
+components:
+  - !NumpyIndexer
+    with:
+      index_filename: vec.gz
+    metas:
+      name: vecidx  # a customized name
+      workspace: $WORKDIR
+  - !BinaryPbIndexer
+    with:
+      index_filename: doc.gz
+    metas:
+      name: docidx
+      workspace: $WORKDIR
```

Comment:
remove workspace inside the `components`, it is a little noisy

Reply:
Should I remove all instances of `$WORKDIR`? There are quite a few in `doc.yml`. How about in other Pods?

Reply:
ping @JoanFM

Reply:
no, u need to remove only those inside the first 2